src/main/java/codeu/controller/LoginServlet.java

```diff
@@ -84,6 +84,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     }
 
     request.getSession().setAttribute("user", username);
+    if(username.equals("lriffle")) {
+    	System.out.print("test");
+    	 request.getSession().setAttribute("admin", "yes");
```

Comment:
Does the attribute have to be a String? If it does not, in my opinion a Boolean would be better for this attribute.